ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/recovery/RecoveryManager.java

```diff
@@ -223,6 +226,10 @@ public RecoveryManagerStatus trySuspend(boolean async) {
         }
     }
 
+    public RecoveryManagerStatus trySuspend(boolean async) {
```

Comment:
I think we need a JavaDoc for this public method (I reason it didn't have one before so we can't really say for this method it's a blocker for this one - unless one could argue the introduction of an optional parameter has made it a bit more necessary?)

Reply:
Done. Thanks Tom

Reply:
IIUC, [this](https://github.com/jbosstm/narayana/pull/2359/commits/e5b865b906fbc449e68e43dccb720559ea81c617#diff-7a8476d3b8fa1bb1cb151e68fc1fef8098f622cd1bb51a6f1471f9e37217b0d5R207-R214) should be the javadoc we were after

Reply:
Here I actually referring to `trySuspend` (thinking along the lines of it being a new method, but I realise it is just moved down from above and that it already didn't have a JavaDoc).

I added my approval because basically though you added a new version of this method, the previous method didn't have a JavaDoc. That being said really both methods should now have a JavaDoc as they are public API methods :)